### ISSUE TITLE
Does this work with offline persistence?

### ISSUE THREAD

**Issue by TheAussieStew** [invertase/react-query-firebase#15]
I searched the repo but couldn't find anything.

**Comment by Ehesp** [comment]
Yep it should do, the library wraps the Firebase SDKs, and the instance you provide can have the network disabled or enabled as explained [here](https://firebase.google.com/docs/firestore/manage-data/enable-offline)
